SU-SWS/stanford_components
# Removed old testing components.

# READY FOR REVIEW

# Summary
- Removes components that were not created for the Earth project.

# Needed By (Date)
- Whenever.

# Urgency
- Not urgent in the least. 

# Steps to Test

1. check out this branch
2. clear site caches
3. review /patterns to ensure simple cta, simple cta with image, and hero with overlay are gone.

# Affected Projects or Products
- Not really.

# Associated Issues and/or People
- @slowbot 
- @koppieesq 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
